lib/src/Base/Graph/DrawableImplementation.cxx

```diff
@@ -924,6 +927,61 @@ Point DrawableImplementation::ConvertFromHSVIntoRGB(const Scalar hue,
   return redGreenBlue;
 }
 
+/* Convert a RGB triplet to HSV */
+Point DrawableImplementation::ConvertFromRGBIntoHSV(const Scalar red,
+    const Scalar green,
+    const Scalar blue)
+{
+  if (red < 0.0 || red > 1.0) throw InvalidArgumentException(HERE) << "Invalid red="<<red;
+  if (green < 0.0 || green > 1.0) throw InvalidArgumentException(HERE) << "Invalid green="<<green;
+  if (blue < 0.0 || blue > 1.0) throw InvalidArgumentException(HERE) << "Invalid blue="<<blue;
+  return ConvertFromRGBIntoHSV(static_cast<UnsignedInteger>(round(255 * red)),
+                               static_cast<UnsignedInteger>(round(255 * green)),
+                               static_cast<UnsignedInteger>(round(255 * blue)));
+}
+/* Convert a RGB triplet to HSV */
+Point DrawableImplementation::ConvertFromRGBIntoHSV(const UnsignedInteger red,
+    const UnsignedInteger green,
+    const UnsignedInteger blue)
+{
+  if (red > 255) throw InvalidArgumentException(HERE) << "Invalid red="<<red;
+  if (green > 255) throw InvalidArgumentException(HERE) << "Invalid green="<<green;
+  if (blue > 255) throw InvalidArgumentException(HERE) << "Invalid blue="<<blue;
+  const Scalar redScalar(red);
```

Comment:
For simple types use affectation, for complex types (classes) use initialization

Reply:
Ok. Done. We may discuss this technical point further in TC.